guide/src/continuous-integration.md

```diff
@@ -0,0 +1,38 @@
+# Continuous integration
```

Comment:
@adamreichold I would highly appreciate some help on this chapter to present some minimal examples for CI usage on these most common providers.

Reply:
Ok, but let's do this as follow-up PR. I think the GitLab CI example is basically sound and it is just a matter of porting it to GH Actions and Travis CI. I also do not think we should include more as these three are already quite redundant.

What would maybe be interesting is an example of how to publish the results, e.g. render the Markdown to HTML using Pandoc and push that to GH Pages?

Reply:
Completely agree on all your points. I did not plan do include more than these three. An example for publishing is definitely a good idea.